frontend/src/pages/Members.vue

```diff
@@ -141,60 +168,65 @@ limitations under the License.
             </v-list-tile-content>
             <v-list-tile-action>
               <v-tooltip top>
-                <v-btn slot="activator" icon class="grey--text" @click.native.stop="onDownload(name)">
+                <v-btn slot="activator" small icon class="blue-grey--text text--darken-2" @click="onKubeconfig(name)">
+                  <v-icon>settings</v-icon>
+                </v-btn>
+                <span>Show Kubeconfig</span>
+              </v-tooltip>
+            </v-list-tile-action>
+            <v-list-tile-action>
+              <v-tooltip top>
+                <v-btn slot="activator" icon class="blue-grey--text text--darken-2" @click.native.stop="onDownload(name)">
                   <v-icon>mdi-download</v-icon>
                 </v-btn>
-                <span>download kubeconfig</span>
+                <span>Download Kubeconfig</span>
               </v-tooltip>
             </v-list-tile-action>
             <v-list-tile-action>
-              <v-btn icon class="red--text text--darken-2" @click.native.stop="onDelete(name)">
-                <v-icon>mdi-delete</v-icon>
-              </v-btn>
+              <v-tooltip top>
+                <v-btn slot="activator" icon class="red--text text--darken-2" @click.native.stop="onDelete(name)">
+                  <v-icon>mdi-delete</v-icon>
+                </v-btn>
+                <span>Delete</span>
```

Comment:
tooltip missing for secrets and maybe rename tooltip to Delete Member / Service Account